examples/objectdetection/src/useDebounce.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export const useDebounce = <T>(value: T, delay: number): T => {
```

Comment:
hmm interesting. This is a somewhat simplified implementation. I am amazed that something so simple would work well.

Reply:
This was mostly implemented by github copilot. Is there something specific you're worried about? I'd prefer to pull in a library if it gets more complex, but didn't see anything obvious (I seem to be failng ag google search on this one).